General Guides writing (#25)
sc2aiur/sc2aiur.github.io: pages/pvp.md

## Welcome to SC2Aiur's general guide to Protoss vs Protoss

> PvP is the only matchup where you will delay your expansion in favor of having more units early. The general game plan starts with two gateways, from which you build either two adepts, two stalkers, or one stalker and a sentry. From there, you can be aggressive or defensive with your starting units, and decide whether to commit to aggression, expand go any of the three tech routes, as robo, twilight into blink or stargate are all viable options. The main mid game compositions to build towards are generally either Chargelot-Immortal-Archon based (CIA), or heavy on the blink stalkers, with disruptors being excellent in either composition.

Reviewer: Here's my first actual content concern:

```suggestion
PvP is the only matchup where you will **delay your expansion in favor of having more units early**. The general game plan starts with **two gateways**, from which you build either two adepts, two stalkers, or one stalker and a sentry. From there, you can be aggressive or defensive with your starting units, and decide whether to commit to aggression, expand, or go any of the three tech routes, as robo, twilight into blink or stargate are all viable options. The main mid game compositions to build towards are generally either Chargelot-Immortal-Archon based (CIA), or one based on [blink stalkers for mobility with disruptors for bursty firepower](posts/blinkdisruptor-era-pvp/).
```

I haven't seen many games with CIA having disruptors mixed in. I'd say it's counterproductive, because your zealots die to your own novas a lot of the time. I'd agree that after an initial period of CIA aggression, a transition to stalker/disruptor is often started. Whatcha think?

### Fundamentals

#### How to make a wall

> In PvP, you want to build a wall at the ramp entering the main, as the two adept opening can one-shot probes and put you very far behind if they can reach there. The wall at the main is generally with a pylon near the ramp, and two gateways leaving a 2 square hole. This hole can be temporarily closed off with a pylon or battery from a probe that is to be left nearby unless you know the opponent is not going for adepts.

Reviewer: This would go great with a screenshot!

#### Scouting
The starting build order for PvP is 14 pylon, 15 gateway, 16 gas, 17gas, 19 gateway, 19 cyber core, 20 pylon. For scouting, you want to use the probe building the second gas at 17 to go down to the natural and explore the edges of it, as to see if there is any proxy pylon placed there. You then bring the probe back to the ramp and build the second gateway. You send a probe to the opponent's base either at this time, or after building the cybernetics core.

> Once arriving at the opponent's main, you want to see the two gateways and cybernetics core building, as well as two pylons in the main, usually one at the ramp and one near his mineral line finishing. The enemy gateway units will spawn at around 2:30 and your probe should have left the base by then, or it will be killed. If you do not see a third pylon in his main, that means he is probably proxying a building, and you may be getting all-inned.

Reviewer: ```suggestion
Upon arriving at the opponent's main, you want to see the two gateways and cybernetics core building, as well as two pylons in the main, usually one at the ramp and one near his mineral line finishing. The enemy gateway units will spawn at around 2:30 and your probe should have left the base by then, or it will be killed. If you do not see a third pylon in his main, that means he is possibly proxying a building, and you may be getting all-inned.

Note that [clever people](https://liquipedia.net/starcraft2/Trap) often proxy just the 3rd pylon somewhere out on the map, as a form of mindgame. This means you often need more information.
```
